dbt/adapters/firebolt/connections.py

```diff
@@ -114,19 +115,21 @@ def get_response(cls, cursor) -> AdapterResponse:
             code=None,
         )
 
-    def begin(self):
+    def begin(self) -> None:
         """
         Passing `SQLConnectionManager.begin()` because
         Firebolt does not yet support transactions.
         """
 
-    def commit(self):
+    def commit(self) -> None:
         """
         Passing `SQLConnectionManager.begin()` because
         Firebolt does not yet support transactions.
         """
 
-    def cancel(self, connection):
+    def cancel(
+        self, connection: SQLConnectionManager
+    ) -> dbt.exceptions.NotImplementedException:
```

Comment:
Does mypy allow this? My issue here is that we're not returning the Exception object, we're raising it. So technically the return type is -> None.

Reply:
Huh. Thought I'd already fixed that.